Common/Tasks/qVectorsCorrection.cxx

```diff
@@ -79,6 +79,9 @@ struct qVectorsCorrection {
   Configurable<float> cfgMaxDCArToPVcut{"cfgMaxDCArToPVcut", 0.1, "Maximum transverse DCA"};
   Configurable<float> cfgMaxDCAzToPVcut{"cfgMaxDCAzToPVcut", 1.0, "Maximum longitudinal DCA"};
 
+  Configurable<int> cfgMaxOccupancy{"cfgMaxOccupancy", 999999, "maximum occupancy of tracks in neighbouring collisions in a given time range"};
```

Comment:
@jikim1290 is the default value large enough not to affect the event selections? I think so, but I would like to have your confirmation

Reply:
Yes it is

Reply:
Perfect, so from my side everything is fine and we can proceed with the approval @ddobrigk